processing/src/main/java/io/druid/query/QueryToolChest.java

```diff
@@ -57,31 +58,36 @@
    *     return new OrderedMergeSequence<>(getOrdering(), seqOfSequences);
    *
    * @param seqOfSequences sequence of sequences to be merged
+   * @param descending order of input and output sequence
    * @return the sequence of merged results
    */
-  public abstract Sequence<ResultType> mergeSequences(Sequence<Sequence<ResultType>> seqOfSequences);
+  public abstract Sequence<ResultType> mergeSequences(Sequence<Sequence<ResultType>> seqOfSequences, boolean descending);
```

Comment:
Technically, this interface is something that someone can extend in an extension, so this change is going to mean that this can't go out until 0.9.0 (which is our next planned release, so no big deal, really).  I say this just to make sure that we tag this PR as 0.9.0 and include something in the release notes about the compatibility change.